force-app/main/default/lwc/ldsUtils/ldsUtils.js

```diff
@@ -19,7 +19,26 @@ export function reduceErrors(errors) {
                     return error.body.map((e) => e.message);
                 }
                 // UI API DML, Apex and network errors
-                else if (error.body && typeof error.body.message === 'string') {
+                else if (error.body && Array.isArray(error.body.message)) {
+                    return error.body.message.map((e) => {
+                        if (e.message) {
+                            return e.message;
+                        } else if (
+                            e.pageErrors &&
+                            Array.isArray(e.pageErrors)
+                        ) {
+                            let pageErrors = [];
+                            e.pageErrors.forEach((pageError) => {
+                                pageErrors.push(pageError.message);
+                            });
+                            return pageErrors.join(' ');
```

Comment:
```suggestion
                            return e.pageErrors.map(pageError => pageError.message).join(' ');
```

Reply:
This is perfect! Thanks @kevinv11n . I am updating it now.